src/dbus/server/introspect.xml

```diff
@@ -235,6 +235,21 @@
       <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
     </property>
 
+    <!-- StartEphemeralKeyMode: Activate ePSKc mode.
```

Comment:
See how this d-bus API is documented: **Activate ePSKc mode.**

If you want to document it as **Activate ePSKc mode.**, then why don't name the API as "ActivateEpskcMode"? @librasungirl

Reply:
Accept, thanks!